flict/flictlib/format/text_format.py

```diff
@@ -20,11 +20,7 @@ def format_support_licenses(self, compatibility):
 
         ret_str = ""
         for item in supported_licenses:
-            lic_group = compatibility.license_group(item)
-            if lic_group is not None:
-                ret_str += " " + str(item) + ": (" + lic_group + ")\n"
-            else:
-                ret_str += " " + str(item) + "\n"
+            ret_str += " " + str(item) + "\n"
```

Comment:
This should at least be an f-string and might be even more simplified.

Reply:
I'll do a rewrite of the strings in entire file afterwards